requirements/base.txt

```diff
@@ -4,6 +4,9 @@
 #
 #    pip-compile --output-file requirements/base.txt requirements/base.in
 #
+--index-url http://edx.devstack.devpi:3141/root/pypi/+simple/
```

Comment:
You should remove these lines before checking in the file. It's a devstack artifact.